### DISCUSSION TITLE
Nginx Proxy Manager Update Request

### DISCUSSION THREAD

**Discussion by Dvalin21** [tteck/Proxmox#3516] [Ideas]
Its it possible to include this with the lxc container in Nginx Proxy Manager https://docs.theme-park.dev/themes/nginx-proxy-manager/

**Comment by Mellowlynx** [comment]
This is how I do it: https://docs.theme-park.dev/setup/#nginx-proxy-manager

**Reply by Dvalin21** [reply]
How did you get this to work and it not cause npm to crash?  Can you share your setup?

**Reply by Mellowlynx** [reply]
I literally did what it says in the link, add custom location `/` and added the code and changed `<APP_NAME>/<THEME>` in the URL. 

![image](https://github.com/user-attachments/assets/075a6b5f-806b-4be9-b4e0-acf73c27ab48)
